src/Features/CSharp/Portable/Completion/CompletionProviders/OverrideCompletionProvider.cs

```diff
@@ -180,7 +179,7 @@ public override bool TryDetermineReturnType(SyntaxToken startToken, SemanticMode
             }
 
             modifiers = new DeclarationModifiers(isUnsafe: isUnsafe, isAbstract: isAbstract, isOverride: true, isSealed: isSealed, isRequired: isRequired);
-            return overrideToken.IsKind(SyntaxKind.OverrideKeyword) && IsOnStartLine(overrideToken.Parent!.SpanStart, text, startLine);
+            return overrideToken.IsKind(SyntaxKind.OverrideKeyword);
```

Comment:
This code was intentional. We didn't want this feature mixed in with some incomplete member above. If we want to support this, we should make sure that it's only attributes above this and nothing else.

Reply:
I made conclusions based on tests and after I removed the second part of the condition no tests failed. So I thought, that this is some kind of outdated code. Can you please provide a test case, where the second part should prevent `override` completion, so I can add it to the suite

Reply:
```
public
override
```